web/src/engine/dom-utils/src/stylesheets.ts

```diff
@@ -29,6 +29,10 @@ export class StylesheetManager {
   }
 
   linkStylesheet(sheet: HTMLStyleElement) {
+    if(!(sheet instanceof HTMLLinkElement) && !sheet.innerHTML) {
+      return;
+    }
```

Comment:
When can this happen?

Reply:
Example keyboard:  `lao_2008_basic` (at least, the local one used in many Web test pages.

The keyboard does not specify either a "display font" or an "osk font", thus requests no keyboard-specific font styling.  This results in one of the standard `VisualKeyboard` stylesheets being completely empty.